res/schema.xml

```diff
@@ -582,4 +582,19 @@ reapplying those migrations.
       UPDATE library SET filetype='aiff' WHERE filetype='aif';
     </sql>
   </revision>
+  <revision version="40" min_compatible="3">
+    <description>
+      Suggestion tables added.
+    </description>
+    <sql>
+      CREATE TABLE IF NOT EXISTS suggestion_library (
+        id INTEGER,
+        artist varchar(48),
+        title varchar(48),
+        playcount INTEGER,
+        match FLOAT,
+        duration FLOAT,
+        location varchar(512));
```

Comment:
What is the use of this column, it contains currently always the same data.

Reply:
I had to add a dummy location in order to not have seg fault.I tried ITunes, the tracks location doesn't exists on my device but it works nice, so I had came up with this solution :sweat_smile:

Reply:
I can imagine to link here an URL of the track.
But the use case is not really clear for me.
Remove?

Reply:
That makes sense IMHO, this can use it to have a specialized suggestion menu maybe? Such as show this on LastFM?

Reply:
I have this assertion when I added URL's as a location `DEBUG ASSERT: "hasLocation(fileInfo)" in function static QString mixxx::FileInfo::canonicalLocation(const QFileInfo&) at /home/whoami/mixxx/src/util/fileinfo``

Should we track it down?

Reply:
Yes, that is easy. Just run Mixxx with GDB and enable `--debug-assert-break`
```
gdb ./mixxx
run --debug-assert-break
   <make it crash> 
bt
``